domain/src/main/kotlin/no/nav/su/se/bakover/domain/grunnlag/Grunnlagsdata.kt

```diff
@@ -28,7 +25,62 @@ data class Grunnlagsdata(
             )
         }
     }
+
+    // TODO jah: Valider at de vurderte uføre-periodene er de samme som de vurderte formue-periodene
+    companion object {
+        val IkkeVurdert = tryCreate()
+        val EMPTY = tryCreate()
+
+        fun tryCreate(
+            fradragsgrunnlag: List<Grunnlag.Fradragsgrunnlag> = emptyList(),
+            bosituasjon: List<Grunnlag.Bosituasjon> = emptyList(),
+        ): Grunnlagsdata {
```

Comment:
Foreslår at vi lager en trello/jira-oppgave på å gjøre om denne returtypen til en either istedet for å kaste exceptions. Jeg ser det er 78-matches på `Grunnlagsdata.tryCreate` så det blir kanskje litt meget i denne PR.